include/manager.h

```diff
@@ -44,25 +46,31 @@ void my_free(void* ptr);
  * Compartment function intercepts
  ******************************************************************************/
 
-// Intercept functions
-time_t manager_time();
-
 struct func_intercept {
     char* func_name;
     uintptr_t redirect_func;
-    void* __capability intercept_pcc;
     void* __capability intercept_ddc;
+    void* __capability intercept_pcc;
+    void* __capability redirect_cap;
 };
 
+void setup_intercepts();
+void print_full_cap(uintcap_t);
+void intercept_wrapper(void* to_call_fn);
+
+// Intercept functions
+time_t manager_time();
 
-static const struct func_intercept comp_intercept_funcs[] = {
+static const struct func_intercept to_intercept_funcs[] = {
     /* vDSO funcs */
     { "time", (uintptr_t) manager_time },
     //"printf",
     /* Mem funcs */
-    { "malloc", (uintptr_t) my_malloc },
-    { "realloc", (uintptr_t) my_realloc },
-    { "free", (uintptr_t) my_free },
+    //{ "malloc", (uintptr_t) my_malloc },
```

Comment:
Should we delete these commented lines?

Reply:
I temporarily commented these out to make my debugging life easier. The next step is to re-add them in and work with them, but this PR is already meaty enough.

Reply:
OK.